app/models/rss_log.rb

```diff
@@ -379,10 +379,10 @@ def detail
     end
     if !target_type
       :rss_destroyed.t(type: :object)
-    elsif !target ||
+    elsif !target_id ||
           tag.to_s.match?(/^log_#{target_type}_(merged|destroyed)/)
       :rss_destroyed.t(type: target_type)
-    elsif !time
+    elsif !time || time < created_at + 1.minute
```

Comment:
Arg, I just realized that this is not quite right.  It used to be (equivalent to) time < target.created_at.  But note that names are auto-created without a log, and their log doesn't get created until the first time they are modified.  The result is, for example, when Joe just updated Macrolepiota procerus your present code will claim that he just *created* it.  When in fact it is name #3702, so it was created way back in 2007.

Consider the following quick fix.  When initializing the log (line 765 of app/models/abstract_model.rb) have it set rss_log.created_at explicitly if the target isn't new:

    rss_log.created_at = created_at unless new_record?

This will ensure that rss_log.created_at really does reflect when the target was created (as your code expects).  Perhaps this is misleading, because the created_at column generally would refer to the time the rss_log was created, but in this case it will instead refer to the time when the *target* was created, not when the rss_log was created.  But I think we can make an exception in this case, no.  What do you think?

Reply:
I like it 👍